datacube/api/core.py

```diff
@@ -277,18 +289,20 @@ def load(self, product=None, measurements=None, output_crs=None, resolution=None
             for more information.
 
         :param xarray.Dataset like:
-            Uses the output of a previous ``load()`` to form the basis of a request for another product.
+            Use the output of a previous ``datacube.load()`` to load data into the same spatial grid and 
+            resolution (i.e. ``.GeoBox``).
             E.g.::
 
                 pq = dc.load(product='ls5_pq_albers', like=nbar_dataset)
 
         :param str group_by:
-            When specified, perform basic combining/reducing of the data.
+            When specified, perform basic combining/reducing of the data. For example, ``group_by='solar_day'``
```

Comment:
Are there other options for this and can we list them?

Reply:
I'm not personally aware of any, but I agree, it would be fantastic to document them if they exist!

Reply:
The other _valid_ value for now is "time" (the default, which does not group.